### DISCUSSION TITLE
Hey, all, is bastion still alive ?

### DISCUSSION THREAD

**Discussion by phil-skillwon** [bastion-rs/bastion#347] [Q&A]
This project hasn't been updated for several years. Is anyone still maintaining it?

**Comment by o0Ignition0o** [comment]
Hey there,

The repo has indeed not been updated in a while, I tend to land bug fixes but I don't see any major improvements or overhauls to bastion happen in the near future.

Happy to discuss with you if you're missing anithing though!

**Reply by phil-skillwon** [reply]
I want to use bastion for some cool projects, but I couldn't find features related to actor clustering or remote actors in bastion, similar to what C++ CAF offers. I'm curious about your thoughts on these features.
